exercises/poker/example.go

```diff
@@ -194,7 +194,7 @@ func isStraight(counts []rankCount) bool {
 	}
 	first := counts[0].rank
 	if first == Ace {
-		for i := 1; i < len(counts); i++ {
+		for i := 1; i < len(counts); i++ { //nolint: staticcheck
```

Comment:
What is `staticcheck` complaining about here? The start from 1??

Reply:
not updating loop counter, false positive - need to report.